lib/optimizely/helpers/event_tag_utils.rb

```diff
@@ -64,55 +66,54 @@ def get_revenue_value(event_tags)
       def get_numeric_value(event_tags, logger = nil)
         # Grab the numeric event value from the event tags. "value" is a reserved keyword.
         # The value of 'value' can be a float or a numeric string
-        # 
+        #
         # event_tags - +Hash+ representing metadata associated with the event.
         # Returns  +Number+ | +nil+ if value can't be retrieved from the event tags.
         logger = SimpleLogger.new if logger.nil?
 
         if event_tags.nil?
-          logger.log(Logger::DEBUG,"Event tags is undefined.")
+          logger.log(Logger::DEBUG, 'Event tags is undefined.')
           return nil
         end
 
-        if !Helpers::Validator.event_tags_valid?(event_tags)
-          logger.log(Logger::DEBUG,"Event tags is not a dictionary.")
+        unless Helpers::Validator.event_tags_valid?(event_tags)
+          logger.log(Logger::DEBUG, 'Event tags is not a dictionary.')
           return nil
         end
 
-        if !event_tags.has_key?(NUMERIC_EVENT_METRIC_NAME)
-          logger.log(Logger::DEBUG,"The numeric metric key is not defined in the event tags.")
+        unless event_tags.key?(NUMERIC_EVENT_METRIC_NAME)
+          logger.log(Logger::DEBUG, 'The numeric metric key is not defined in the event tags.')
           return nil
         end
 
         if event_tags[NUMERIC_EVENT_METRIC_NAME].nil?
-          logger.log(Logger::DEBUG,"The numeric metric key is null.")
+          logger.log(Logger::DEBUG, 'The numeric metric key is null.')
           return nil
         end
 
         raw_value = event_tags[NUMERIC_EVENT_METRIC_NAME]
 
-        if raw_value === true or raw_value === false
-          logger.log(Logger::DEBUG,"Provided numeric value is a boolean, which is an invalid format.")
+        if raw_value.is_a?(TrueClass) || raw_value.is_a?(FalseClass)
```

Comment:
Is this more performant? I am wondering if we really need to make this particular change

Reply:
http://rubocop.readthedocs.io/en/latest/cops_style/?highlight=CaseEquality#stylecaseequality
I think it's more readable this way. But if you think otherwise, I can disable this cop in rubocop.yml